homeassistant/components/light/hue.py

```diff
@@ -397,4 +408,25 @@ def device_state_attributes(self):
         attributes = {}
         if self.is_group:
             attributes[ATTR_IS_HUE_GROUP] = self.is_group
+        if self.gamut_typ:
+            attributes[ATTR_GAMUT_TYP] = self.gamut_typ
+        if self.gamut:
+            attributes[ATTR_GAMUT] = self.gamut
```

Comment:
True they can never change, but I wanted a way to lookup the color gamut inside the frontend of homeassistant.
Now you can see them under the dev states panel.

Reply:
State attributes are for dynamic info related to the state of the entity.

Reply:
@MartinHjelmare alright, do you know any other place where I can put this info that is accessible through the front end?
Although things like the `supported_features`, `friendly_name`, `effect_list`, `min_mireds`, `max_mireds` and `is_hue_group` are all vissable in the dev states panel for a Hue light, therefore I think those are also state attributes and those are also not dynamic but just set once when HomeAssistant starts.

Reply:
There will always be exceptions to a rule. It doesn't mean the rule should be broken or is bad. And rules evolve over time. All parts of the source don't conform to current standards and rules.

`device_info` property would be appropriate in my opinion, but as far as I know, we can't add any key, but need to standardize those. Ask about this in discord chat, and if we need to standardize, it will require an architecture issue discussion and approval.

Reply:
@MartinHjelmare I wil just remove this part of the code for now.
It is not essential for opperation, just nice to be able to see what kind of lights you have.